src/css/Favorites.css

```
@@ -0,0 +1,194 @@
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css");
+
+body {
+  font-family: "Pretendard Variable";
+}
+nav {
+  display: flex;
+  justify-content: center;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  background-color: #edf7ff;
+}
+
+.gnb {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 93px;
+  max-width: 192rem;
+  padding: 0 200px;
+}
+
+.cta {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 5.4rem;
+  cursor: pointer;
+  background-image: linear-gradient(135deg, var(--primary) 0%, #6ae3fe 100%);
+  border-radius: 0.8rem;
+  color: #f5f5f5;
+  font-size: 1.8rem;
+  font-weight: 600;
+}
+
+.cta-short {
+  width: 12.8rem;
+}
+
+.FavoritesProfileContent {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.FavoritesProfileImgSrc {
+  width: 28px;
+  height: 28px;
+  border-radius: 100px;
+  margin-right: 6px;
+}
+
+.FavoritesProfileEmail {
+  text-decoration: none;
+  color: black;
+}
+
+.FavoritesProfile {
+  height: 244px;
+  background-color: #f0f6ff;
+  padding: 20px 624px 60px 624px;
```

컨텐츠를 중앙정렬 하실 의도로 좌우 패딩을 많이 주신것 같아요. 이렇게 구현하기 보다 flex의 `justify-content` 속성으로 정렬하는게 좋을 것 같습니다!
```suggestion
  padding: 20px 0 60px 0;
  justify-content: center;
```